Dfe.Academies.External.Web/CypressTests/cypress/e2e/createNewJAMApp.cy.ts

```diff
@@ -0,0 +1,1079 @@
+        Footer.cookiesLinkVisible()
+        Footer.termsAndConditionsLinkVisible()
+        Footer.privacyLinkVisible()
+        Footer.oglLogoVisible()
+        Footer.allContentTextVisible()
+        Footer.openGovernmentLicence3LinkVisible()
+        Footer.crownCopyrightLinkVisible()
+
+        A2BYourApplications.selectStartANewApplication()
+
+        Header.govUkHeaderVisible()
+        Header.applyToBecomeAnAcademyHeaderLinkVisible()
+
+        A2BWhatAreYouApplyingToDo.whatAreYouApplyingToDoElementsVisible()
+
+        Footer.accessibilityStatementLinkVisible()
+        Footer.cookiesLinkVisible()
+        Footer.termsAndConditionsLinkVisible()
+        Footer.privacyLinkVisible()
+        Footer.oglLogoVisible()
+        Footer.allContentTextVisible()
+        Footer.openGovernmentLicence3LinkVisible()
+        Footer.crownCopyrightLinkVisible()
+
+        A2BWhatAreYouApplyingToDo.selectJAMRadioButton()
+        A2BWhatAreYouApplyingToDo.verifyJAMRadioButtonChecked()
+
+        A2BWhatAreYouApplyingToDo.selectApplyingToDoSaveAndContinue()
+
+        Header.govUkHeaderVisible()
+        Header.applyToBecomeAnAcademyHeaderLinkVisible()
+
+        A2BWhatIsYourRole.whatIsYourRoleElementsVisible()
+
+        Footer.accessibilityStatementLinkVisible()
+        Footer.cookiesLinkVisible()
+        Footer.termsAndConditionsLinkVisible()
+        Footer.privacyLinkVisible()
+        Footer.oglLogoVisible()
+        Footer.allContentTextVisible()
+        Footer.openGovernmentLicence3LinkVisible()
+        Footer.crownCopyrightLinkVisible()
+
+        A2BWhatIsYourRole.selectChairOfGovernorsRadioButton()
+        A2BWhatIsYourRole.verifyChairOfGovernorsRadioButtonChecked()
+
+        A2BWhatIsYourRole.selectWhatIsYourRoleSaveAndContinue()
+
+        // VERIFY JAM YOUR APPLICATION OVERVIEW PAGE DISPLAYS CORRECTLY
+        Header.govUkHeaderVisible()
+        Header.applyToBecomeAnAcademyHeaderLinkVisible()
+
+        A2BYourApplication.yourApplicationNotStartedElementsVisible()
+   
+        Footer.accessibilityStatementLinkVisible()
+        Footer.cookiesLinkVisible()
+        Footer.termsAndConditionsLinkVisible()
+        Footer.privacyLinkVisible()
+        Footer.oglLogoVisible()
+        Footer.allContentTextVisible()
+        Footer.openGovernmentLicence3LinkVisible()
+        Footer.crownCopyrightLinkVisible()
+
+        //Click Add a Trust
+        A2BYourApplication.selectAddATrust()
+
+       Header.govUkHeaderVisible()
+       Header.applyToBecomeAnAcademyHeaderLinkVisible()
+
+       A2BWhichTrustIsSchoolJoining.whichTrustIsSchoolJoiningElementsVisible()
+
+       Footer.accessibilityStatementLinkVisible()
+       Footer.cookiesLinkVisible()
+       Footer.termsAndConditionsLinkVisible()
+       Footer.privacyLinkVisible()
+       Footer.oglLogoVisible()
+       Footer.allContentTextVisible()
+       Footer.openGovernmentLicence3LinkVisible()
+       Footer.crownCopyrightLinkVisible()
+
+       cy.wait(2000)
```

Comment:
I'd avoid explicit waits as much as possible, instead opting for an implicit wait

See docs [here](https://docs.cypress.io/api/commands/wait#Time)